src/mysql.js

```diff
@@ -4,7 +4,8 @@ const connection = mysql.createConnection({
 	host: process.env.DB_HOST,
 	user: process.env.DB_USER,
 	password: process.env.DB_PASS,
-	database: process.env.DB_NAME
+	database: process.env.DB_NAME,
+	multipleStatements: true
```

Comment:
Execute queries with multiple statements by separating each statement with a semi-colon `;`
https://stackoverflow.com/questions/23266854/node-mysql-multiple-statements-in-one-query